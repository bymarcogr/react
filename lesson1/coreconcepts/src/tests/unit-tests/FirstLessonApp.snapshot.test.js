import renderer from 'react-test-renderer';
import FirstLessonApp from '../../components/firstLessonApp';

it('renders correctly', () => {
    const tree = renderer
      .create(<FirstLessonApp />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });