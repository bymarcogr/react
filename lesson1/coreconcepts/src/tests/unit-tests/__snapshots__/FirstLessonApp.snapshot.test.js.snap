// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "padding": "5px",
    }
  }
>
  <h1
    className="title"
  >
    Counter Component
  </h1>
  <span>
    <span
      className="counter-title"
      title="number-to-display"
    >
      1
    </span>
    <br />
    <span>
      <button
        className="genericButton"
        onClick={[Function]}
      >
        -
      </button>
    </span>
    <span>
      <button
        className="genericButton"
        onClick={[Function]}
      >
        +
      </button>
    </span>
  </span>
  <h1
    className="title"
  >
    Search Component
  </h1>
  <br />
  <span>
    <input
      className="search"
      onChange={[Function]}
      onKeyDown={[Function]}
      placeholder="What do you want to watch?"
      type="text"
      value="Search String?"
    />
    <span>
      <button
        className="netflixSearch"
        onClick={[Function]}
      >
        Search
      </button>
    </span>
  </span>
  <h1
    className="title"
  >
    Genre List Component
  </h1>
  <br />
  <span>
    <span>
      <button
        className="netflixBar"
        onClick={[Function]}
      >
        Action
      </button>
    </span>
    <span>
      <button
        className="netflixBar"
        onClick={[Function]}
      >
        Comedy
      </button>
    </span>
    <span>
      <button
        className="netflixBar"
        onClick={[Function]}
      >
        Drama
      </button>
    </span>
    <span>
      <button
        className="netflixBarSelected"
        onClick={[Function]}
      >
        Fantasy
      </button>
    </span>
    <span>
      <button
        className="netflixBar"
        onClick={[Function]}
      >
        Horror
      </button>
    </span>
    <span>
      <button
        className="netflixBar"
        onClick={[Function]}
      >
        Mystery
      </button>
    </span>
    <span>
      <button
        className="netflixBar"
        onClick={[Function]}
      >
        Romance
      </button>
    </span>
    <span>
      <button
        className="netflixBar"
        onClick={[Function]}
      >
        Thriller
      </button>
    </span>
    <span>
      <button
        className="netflixBar"
        onClick={[Function]}
      >
        Western
      </button>
    </span>
  </span>
</div>
`;
